src/Crud/PagesRequest.php

```diff
@@ -0,0 +1,35 @@
+<?php
+declare(strict_types=1);
+
+namespace Biig\Melodiia\Crud;
+
+class PagesRequest
```

Comment:
A new subnamespace `Pagination` looks accurate. (because pagination is a real thing)

Reply:
I hesitate, then I saw no namespace for filters (and it's a real thing too 👼)